src/components/TimeSeriesCard/timeSeriesUtils.js

```diff
@@ -116,18 +117,22 @@ export const formatGraphTick = (
     ? currentTimestamp.format(dailyFormat)
```

Comment:
This is a ton of conditionals, but I'm not seeing a better way at the moment. Is it possible to only run some of  these checks if the language is Chinese?

Reply:
I could switch statement and work within there, but honestly I'm just changing the existing code to support week.